README.md

```diff
@@ -63,3 +66,141 @@ To use the **.NET** version of the schema add the `MicroFocus.FAS.AdapterSdkSche
 - Check the field's type:  
 
     `AdapterSdkSchema.GetField("TITLE").FieldType;`
+
+Create a **Java** `AdapterSdkSchemaObject` using the `AdapterSdkSchemaObjectBuilder`
+
+- Create a `AdapterSdkSchemaObjectBuilder` passing in an instance of a `SchemaObjectBuilder`
+
+```
+    final Map<String, Object> document = new HashMap<String, Object>();
+    final AdapterSdkSchemaObjectBuilder documentBuilder
+        = new AdapterSdkSchemaObjectBuilder(new MapSchemaObjectBuilder(document));
+```
+
+- Set field values by calling the appropriate setter methods
+
+```
+    documentBuilder.setFileName("test.doc");
+    documentBuilder.setMimetype("msword");
+
+    documentBuilder.setAddressDisplayBcc("person1@abc.com", "person2@abc.com");
+    documentBuilder.setAddressDisplayTo("johnDoe@abc.com");
+```
+
+- Clear a field
+
+```
+    documentBuilder.clearAddressDisplayBcc();
+```
+
+- Add more values to a field
+
+```
+    documentBuilder.addAddressDisplayTo("janeDoe@abc.com");
+```
+
+- Check if all mandatory fields have been set and a valid `AdapterSdkSchemaObject` has been created
+
+```
+    documentBuilder.validate();
+```
+
+Create a **.NET** `AdapterSdkSchemaObject` using the `AdapterSdkSchemaObjectBuilder`
+
+- Create a `AdapterSdkSchemaObjectBuilder` passing in an instance of a `ISchemaObjectBuilder`
+
+```
+    var document = new Dictionary<string, object>();
+    AdapterSdkSchemaObjectBuilder documentBuilder = new(new DictionarySchemaObjectBuilder(document));
+```
+
+- Set field values by calling the appropriate setter methods
+
+```
+    documentBuilder.SetFileName("test.doc");
+    documentBuilder.SetMimetype("msword");
+
+    documentBuilder.SetAddressDisplayBcc("person1@abc.com", "person2@abc.com");
+    documentBuilder.SetAddressDisplayTo("johnDoe@abc.com");
+```
+
+- Clear a field
+
+```
+    documentBuilder.ClearAddressDisplayBcc();
+```
+
+- Add more values to a field
+
+```
+    documentBuilder.AddAddressDisplayTo("janeDoe@abc.com");
+```
+
+- Check if all mandatory fields have been set and a valid `AdapterSdkSchemaObject` has been created
+
+```
+    documentBuilder.Validate();
+```
+
+The SchemaObjectBuilder encodes field values as **json** or **flattened**
```

Comment:
It's really an internal detail when you're using the object builder, which is sort of the point.  I think it would be more useful to show the usage than to show the output.

Reply:
Added usage